### PR TITLE
*: batch write for admin pause 

### PR DESCRIPTION
**Pull request by hawkingrei**
### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tidb/issues/45906

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: ddl/tests/adminpause/ddl_data_generation.go [hunk 1]
```diff
@@ -121,36 +121,36 @@ func (tu *TestTableUser) generateAttributes(id int) (err error) {
 	return nil
 }
 
-func (tu *TestTableUser) insertStmt(tableName string) string {
-	return fmt.Sprintf("INSERT INTO %s(tenant, name, age, province, city, phone, created_time, updated_time) "+
-		"VALUES ('%s', '%s', %d, '%s', '%s', '%s', '%s', '%s')",
-		tableName, tu.tenant, tu.name, tu.age, tu.province, tu.city, tu.phone, tu.createdTime, tu.updatedTime)
+func (tu *TestTableUser) insertStmt(tableName string, count int) string {
+	sql := fmt.Sprintf("INSERT INTO %s(tenant, name, age, province, city, phone, created_time, updated_time) VALUES ", tableName)
+	for n := 0; n <= count; n++ {
```
Comment (Defined2014) [inline ddl/tests/adminpause/ddl_data_generation.go:126]: looks like one more loop than before

File: ddl/tests/adminpause/ddl_data_generation.go [hunk 2]
```diff
@@ -121,36 +121,36 @@ func (tu *TestTableUser) generateAttributes(id int) (err error) {
...
-	for idx := 0; idx < rowCount; {
-		_ = tu.generateAttributes(idx)
-		tk.MustExec(tu.insertStmt(adminPauseTestPartitionTable))
-
-		idx++
+	if rowCount == 0 {
+		return nil
 	}
+	tu := &TestTableUser{}
+	tk.MustExec(tu.insertStmt(adminPauseTestPartitionTable, rowCount))
```
Comment (Defined2014) [inline ddl/tests/adminpause/ddl_data_generation.go:153]: Untouched 🥲

File: ddl/tests/adminpause/ddl_data_generation.go (hunk 1 above)
Comment (hawkingrei) [reply ddl/tests/adminpause/ddl_data_generation.go]: Done

File: ddl/tests/adminpause/ddl_data_generation.go (hunk 2 above)
Comment (hawkingrei) [reply ddl/tests/adminpause/ddl_data_generation.go]: It is useless. removed.

File: ddl/tests/adminpause/ddl_data_generation.go [hunk 3]
```diff
@@ -121,36 +121,30 @@ func (tu *TestTableUser) generateAttributes(id int) (err error) {
...
-	return fmt.Sprintf("INSERT INTO %s(tenant, name, age, province, city, phone, created_time, updated_time) "+
-		"VALUES ('%s', '%s', %d, '%s', '%s', '%s', '%s', '%s')",
-		tableName, tu.tenant, tu.name, tu.age, tu.province, tu.city, tu.phone, tu.createdTime, tu.updatedTime)
+func (tu *TestTableUser) insertStmt(tableName string, count int) string {
+	sql := fmt.Sprintf("INSERT INTO %s(tenant, name, age, province, city, phone, created_time, updated_time) VALUES ", tableName)
+	for n := 0; n < count; n++ {
+		_ = tu.generateAttributes(n)
+		sql += fmt.Sprintf("('%s', '%s', %d, '%s', '%s', '%s', '%s', '%s')",
+			tu.tenant, tu.name, tu.age, tu.province, tu.city, tu.phone, tu.createdTime, tu.updatedTime)
+		if n != count {
```
[the remark this answers is not in the document]
Comment (hawkingrei) [reply ddl/tests/adminpause/ddl_data_generation.go:130]: Sure. fixed.

**Pull request merged**